frontend2/package.json

```diff
@@ -17,6 +17,7 @@
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.45.1",
     "react-router-dom": "^6.13.0",
+    "react-markdown": "^8.0.7",
```

Comment:
in the future, we may consider switching to `marked` or `remarkable`, because they are about 1/3 the size of react-markdown.
we'll just need to figure out how to replace `<a>`s that link to urls within play.batlecode.org with `<Link>`